common/persistence/sql/sqlExecutionManager.go

```diff
@@ -1076,6 +1076,11 @@ func updateBufferedEvents(tx *sqlx.Tx, batch *p.DataBlob, clear bool, shardID in
 	return nil
 }
 
+func (m *sqlExecutionManager) ResetWorkflowExecution(request *p.InternalResetWorkflowExecutionRequest) error {
+	// TODO need to implement with history eventsV2 API
+	return fmt.Errorf("Not implemented")
```

Comment:
Can you file a task to implement this later?

Reply:
OK https://github.com/uber/cadence/issues/1368